youtube_dl/extractor/letv.py

```diff
@@ -239,3 +240,75 @@ class LetvPlaylistIE(LetvTvIE):
         },
         'playlist_mincount': 7
     }]
+
+
+class LetvCloudIE(InfoExtractor):
+    IE_DESC = 'LetvCloud'
+    _VALID_URL = r'http://yuntv\.letv\.com/bcloud.html\?(?P<id>.*)$'
```

Comment:
The group ```id``` is not for IDs but URL parameters. This can be misleading, and I suggest a different name.